File: javascript/challenge12.js
const checkAir = (samples,threshold) => {
    let dirtyCount = 0;
    samples.forEach((v) => (v === 'dirty' && dirtyCount++));  
    if (threshold >= dirtyCount/samples.length){
      return 'Clean'
    } else {
      return 'Polluted'
    }
}

File: python/challenge12.py
def checkAir(samples, threshold):
    dirtyCount = samples.count("dirty")
    if (threshold >= dirtyCount/len(samples)):
      return "Clean"
    else:
      return "Polluted"
      
# samples = ['clean', 'clean', 'dirty', 'clean', 'dirty', 'clean', 'clean', 'clean', 'clean', 'clean']
# threshold = 0.3            
# checkAir(samples, threshold)
